tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from unittest.mock import Mock
+from scene_select.utils import calc_file_path
+
+
+def test_local_path():
+    # s2
+    # uris = ["zip:/yada/yada/yada20124T021536.zip!/"]
+
+    # ls
+    # "local_path": "PosixPath('/g/data/u46/yada_01_T2.odc-metadata.yaml')
+    s2_l1_dataset = Mock()
+    s2_l1_dataset.local_path = None
+    path = "/g/S2A_MSIL1C_20220124T004711_N0301_R102_T54LYH_20220124T021536.zip"
+    s2_l1_dataset.uris = ["zip:" + path + "!/"]
```

Comment:
recommendation: might be safer to concat the uri path elements using urllib.parse.urljoin()

Reply:
Thanks. Done.

Reply:
This is more testing the string, so urllib.parse.urljoin() wouldn't work.